src/pages/studyView/StudyViewPage.tsx

```diff
@@ -377,6 +377,14 @@ export default class StudyViewPage extends React.Component<
         }
     }
 
+    @computed get shouldShowResourcesTab() {
+        if (this.store.resourceDefinitions.isComplete) {
+            return this.store.resourceDefinitions.result.length > 0;
+        } else {
+            return false;
+        }
+    }
+
     @computed get shouldShowResources() {
```

Comment:
It was used for showing the tab originally, also, there is some old code in the Resource Tab that might need to be used in the future. However, I will remove it according to your comment, if it is needed in the future, it can be added easily.